### DISCUSSION TITLE
Windows 11 theme

### DISCUSSION THREAD

**Discussion by Pixel556** [DustinBrett/daedalOS#132] [Ideas]
Hey, I was wondering if you could make another website not with Windows 10 look, but Windows 11 look. The website should be: https://dustinbrett.com/#win11. If you don't want to that's fine

**Comment by DustinBrett** [comment]
Hi, thanks for the suggestion! This project is planned as a life long endeavor. I plan to eventually evolve the theming to be very flexible. As for Windows 11 theme specifically, I've been planning to do that as soon as Microsoft convinces me to upgrade my local machine. Currently my preference is Windows 10 and I couldn't do a Windows 11 theme properly unless I went all in.

**Reply by nequals** [reply]
Actually, you could use [this](https://win11.blueedge.me/) to get an idea of what it looks like

**Reply by DustinBrett** [reply]
Thanks for the suggestion but when I do move to Windows 11 I will base any theme on the latest build of the OS.

**Comment by Pixel556** [comment]
@ZJUSTKIDDING I used that before.
